modules/database/src/adapters/mongoose-adapter/MongooseSchema.ts

```diff
@@ -58,7 +57,7 @@ export class MongooseSchema extends SchemaAdapter<Model<any>> {
   parseStringToQuery(
     query: Query | SingleDocQuery | MultiDocQuery,
   ): ParsedQuery | ParsedQuery[] {
-    return typeof query === 'string' ? EJSON.parse(query) : query;
+    return typeof query === 'string' ? JSON.parse(query) : query;
```

Comment:
How does this handle Dates and Mongo ObjectIDs? Both are probably still strings at this point?